docs/using-a-listview.md

```diff
@@ -89,7 +89,7 @@ const SectionListBasics = () => {
           ]}
           renderItem={({item}) => <Text style={styles.item}>{item}</Text>}
           renderSectionHeader={({section}) => <Text style={styles.sectionHeader}>{section.title}</Text>}
-          keyExtractor={(item, index) => index}
+          keyExtractor={(item, index) => `${item}-${index}`}
```

Comment:
Hi @Shubh1902, thank you for sotting this out! 👍 

Unfortunately this change might also cause an issue or be at least confusing, when the data list includes objects not strings. Storing objects seems to be the most common approach to providing the list data, see:
* https://reactnative.dev/docs/flatlist#keyextractor

What do you think about replacing reference to`${item}` with a hardcoded string for this example?